### PR TITLE
OBAM-337 Show returns as returns in stock history

### PR DESCRIPTION
**Pull request by hdsldv**

### CODE REVIEW

File: grails-app/views/inventoryItem/_showStockHistory.gsp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<%@ page import="org.pih.warehouse.product.Product"%>
+<%@ page import="org.pih.warehouse.order.OrderTypeCode; org.pih.warehouse.product.Product"%>
```
Comment (awalkowiak) [inline grails-app/views/inventoryItem/_showStockHistory.gsp:1]: Split it into two lines

File: grails-app/views/inventoryItem/_showStockHistory.gsp [hunk 2]
```diff
@@ -340,6 +340,26 @@
 
                                 <g:if test="${stockHistoryEntry?.showDetails}">
                                     <div >
+                                        <g:if test="${stockHistoryEntry?.transaction?.order?.code == org.pih.warehouse.order.OrderTypeCode.PURCHASE_ORDER.name()}">
```
Comment (awalkowiak) [inline grails-app/views/inventoryItem/_showStockHistory.gsp:343]: You are importing `OrderTypeCode` at the top. `OrderTypeCode.PURCHASE_ORDER.name()` should work.

File: grails-app/views/inventoryItem/_showStockHistory.gsp [hunk 3]
```diff
@@ -340,6 +340,26 @@
...
+                                            <g:link controller="stockMovement" action="show" id="${stockHistoryEntry?.transaction?.order?.id }">
+                                                <div title="${stockHistoryEntry?.transaction?.order?.name }">
+                                                    <format:metadata obj="${stockHistoryEntry?.transaction?.order?.orderType?.code }"/>
+                                                    &rsaquo;
+                                                    ${stockHistoryEntry?.transaction?.order?.orderNumber }
+                                                </div>
+                                            </g:link>
+                                        </g:if>
+
+                                        <g:elseif test="${stockHistoryEntry?.transaction?.order?.code == org.pih.warehouse.core.Constants.RETURN_ORDER}">
```
Comment (awalkowiak) [inline grails-app/views/inventoryItem/_showStockHistory.gsp:353]: You can import `Constants` too, same as `OrderTypeCode`

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: grails-app/views/inventoryItem/_showStockHistory.gsp (hunk 3 above)
Comment (hdsldv) [reply grails-app/views/inventoryItem/_showStockHistory.gsp]: done

File: grails-app/views/inventoryItem/_showStockHistory.gsp (hunk 2 above)
Comment (hdsldv) [reply grails-app/views/inventoryItem/_showStockHistory.gsp]: done

File: grails-app/views/inventoryItem/_showStockHistory.gsp (hunk 1 above)
Comment (hdsldv) [reply grails-app/views/inventoryItem/_showStockHistory.gsp]: done

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**